stock_analytic/tests/test_stock_picking.py

```diff
@@ -12,7 +12,7 @@
 from odoo.tests.common import TransactionCase
 
 
-class TestStockPicking(TransactionCase):
+class CommonStockPicking(TransactionCase):
```

Comment:
Please put it in a separate `common.py` file

Reply:
Please don't. That makes the blame a hell. It's not technically needed anyway, and I doubt its usefulness as organizational element.